environmentsetup/initial-.com-environment-setup.md

```diff
@@ -0,0 +1,67 @@
+1.	Download Visual studio Professional 2017 here: https://www.visualstudio.com/downloads/.
+1.  The Visual Studio installer will direct you to an **Installing Visual Studio** screen. You can select additional components to include with the Visual Studio core editor. Don't deselect any components selected by default.
+1.	Make sure the following are selected under the **Workloads** tab of the **Installing Visual Studio** screen
+    *	.NET desktop development
+    *	Desktop development with C++
+    *	ASP.NET and web development
+    *	Azure development
+    *	.NET Core cross-platform development
+1.	These components should be added under the **Individual Components** tab
+    *	Python 3 64-bit (3.6.3)
+    *	Windows 8.1 SDK
+1. Once the correct components are selected, finish the installation process. 
+   
+### Verify permissions
+
+Once Visual Studio 2017 has been successfully installed, a Visual Studio Team Services (VSTS) user account will need to be associated with it. The following is a way to check if your account has been created and given proper permissions. This section can be skipped if you know the correct permissions have been granted.  
+1.	Go to the GTM Sportswear VSTS website here: https://gtmsportswear.visualstudio.com/_projects.
+1.	Sign in using your work email and associated password.
+1.	Under the **settings** (looks like a gear) dropdown select **Security**. 
+1.	Select **GTM Sportswear – Development Group** in the drop down list on the left hand side of the page.
+1.	Select **Members** and verify that your name is included on the list of members.
+    
+### Clone the Project
+
+Now Open Visual studio 2017. In the top right hand corner of Visual Studio is a prompt to sign in as a user. 
+1.	Sign in using your work email and password.
+1.	If your account has been given proper permissions you should now be able to access GTM projects through the **Team Explorer** window. 
+1.	Select the **green connection icon** at the top of the **Team Explorer** window.
+1.	Click the **manage connections Icon** and select **Connect to a Project** in the dropdown.   
+1.	From the new view you can select the **gtm ->> gtm** project and clone it to a destination of your choice (the default location is fine).
+1.  Once the project has finished downloading, it will be automatically selected and appear in the **Solution Explorer**.
+1.  Double click on **GTMSportswear** in the **Solution Explorer**. This sets it as the default startup project.
+1.  Visual Studio may ask you if you want to update the Typescript version in your project to match the current version. Select **no**. We do not transpile typescript through Visual Studio, we instead use seperate build tools (setup below) through Window's Command Prompt.
+
+### Remove Gtm.Web from Visual Studios Build List

```

Comment:
I think this might be due to an issue with the .NET SDK that's installed on the dev's initial machine setup. I updated my .NET SDK version and don't have to remove that from the build list.

Here's how my configuration manager panel looks:
![image](https://user-images.githubusercontent.com/2092656/34851886-b28ba86a-f6f1-11e7-93cd-9b0f96511483.png)

Reply:
I have updated and checked versioning, and done a restart. It doesn't seem to fix the issue. Do I need to actually remove older versions? It appears to maintain multiple.